quickstart.sh

```diff
@@ -1,17 +1,39 @@
+# Prerequsites
+# 1. GCP billing account name found at https://console.cloud.google.com/billing, in the examples this is `My Billing Account FY22`
+# 2. Desired project ID name, in the examples this is `cvah-helk-training-fy22`
+
+# Install deps
+# curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+# sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+# sudo apt-get update && sudo apt-get install terraform packer
+
 # Create a new project
 pushd ./terraform/setup
-terraform init 
+# Do a plan to ensure configs are set up
+# terraform init -var "gcp_billing_account_name=My Billing Account FY22" -var "project_id=cvah-helk-training-fy22"
```

Comment:
`terraform init` does not require the variables to be passed. All it does is pull down the proper provider package. It doesn't stop anything.